crates/solidity/inputs/language/src/definition.rs

```diff
@@ -5749,11 +5311,6 @@ codegen_language_macros::compile!(Language(
                                 value = Atom("new")
                             )]
                         ),
-                        Keyword(
-                            name = YulNotKeyword,
-                            identifier = YulIdentifier,
-                            definitions = [KeywordDefinition(value = Atom("not"))]
-                        ),
                         Keyword(
```

Comment:
Removing these yul keywords means they are no longer reserved, which can lead us to parse the following incorrectly as an identifier, and produce no errors. Is that a regression?

```
let not := 0
```

Reply:
Oh... I didn't realize, but yes, I think it's a regression. That means we cannot change the grammar as we intended, right? But we'll instead need to add binding rules for all reserved keywords, to generate binding references from them.

Reply:
Or can we parse them as `YulIdentifier` in the context of a function call, while still reserving them as keywords?

Reply:
Not sure if a bigger change is needed, since we are not doing validation.
I would be more concerned about cases where we produce incorrect trees for valid code, which is a simpler problem to solve.
Let's talk about it on Monday before we do a bigger change here.

Reply:
Thinking about it, I'm more concerned about cases where we produce incorrect trees for valid code, which is a simpler problem to solve.

Let's talk about it on Monday before we do a bigger change here, since we are not doing validation.